Makefile

```diff
@@ -56,6 +59,27 @@ update-go-deps:
 .PHONY: verify-go-deps
 verify-go-deps:
 	hack/make-rules/verify/go-deps.sh
+# ======================== Image Building/Publishing ===========================
+# build and publish miscellaneous images that get pushed to gcr.io/k8s-prow
```

Comment:
These new make rules are closely based on the rules in `prow/Makefile`.

Reply:
Nit: Turn line 63 into a regular sentence with a period at the end.